webapp/css/netpyne.less

```diff
@@ -569,3 +568,94 @@ div.flexlayout__layout {
   overflow-y: auto;
   flex: 1;
 }
+//experiment table start
+.experimentTableContainer {
```

Comment:
I would suggest to define styling in JS with `withStyles` of material-ui and use `spacing` and `typography` instead of hardcoded font-weight and px values. 

Netpyne-UI has currently a mix of different ways to define styling, I also see many components using withStyles:
see `ImportCellParams.js`, or `LayoutManager.tsx`. And I suppose `withStyles` was introduced more recently.

I don't know about the styling history of this project, did we settle on a specific approach? (@ddelpiano and @filippomc ) Will update this accordingly in the README